File: Grade program/Grade program.java
import java.io .*;
import java.util.Scanner;

class GFG {
    public static void main(String[] args)
    {
        while (true){
            Scanner num = new Scanner(System.in);
            System. out.println("1. Convert marks.");
            System.out.println("2. Exit.");
            System. out.print("Choose option(1 or 2): ");
            int input = num.nextInt();
            if (input == 1){
                System. out.print("Enter marks: ");
                int marks = num.nextInt();
                if (marks >= 90 && marks <= 100) {
                    System. out. println("The grade for " + marks + " is A.");
                    System.out.println("");
                }
                else if (marks >= 87 && marks <= 89){
                    System. out. println("The grade for " + marks + " is A -. ");
                    System.out.println("");
                }
                else if (marks >= 84 && marks <= 86){
                    System. out. println("The grade for " + marks + " is B+.");
                    System.out.println("");
                }
                else if (marks >= 80 && marks <= 83){
                    System. out. println("The grade for " + marks + " is B.");
                    System.out.println("");
                }
                else if (marks >= 77 && marks <= 79){
                    System. out. println("The grade for " + marks + " is B -. ");
                    System.out.println("");
                }
                else if (marks >= 74 && marks <= 76){
                    System. out. println("The grade for " + marks + " is C+.");
                    System.out.println("");
                }
                else if (marks >= 70 && marks <= 73){
                    System. out. println("The grade for " + marks + " is C.");
                    System.out.println("");
                }
                else if (marks >= 67 && marks <= 69){
                    System. out. println("The grade for " + marks + " is C -. ");
                    System.out.println("");
                }
                else if (marks >= 64 && marks <= 66){
                    System. out. println("The grade for " + marks + " is D+.");
                    System.out.println("");
                }
                else if (marks >= 62 && marks <= 63){
                    System. out. println("The grade for " + marks + " is D.");
                    System.out.println("");
                }
                else if (marks >= 60 && marks <= 61){
                    System. out. println("The grade for " + marks + " is D -. ");
                    System.out.println("");
                }
                else if (marks >= 0 && marks <= 59){
                    System. out. println("The grade for " + marks + " is F.");
                    System.out.println("");
                }
                else{
                    System.out.println("Invalid input!");
                    System.out.println("");
                }

            else if (input == 2){
                System. out.println("Goodbye!");
                break;
            }
            else{
                System.out.println("Invalid input.");
                System.out.println("");  }
            }
        }
    }
}

File: Grade program/Grade program.py
while True:
    print("1. Convert marks.")
    print("2. Exit.")
    uinput = int(input("Choose option (1 or 2): "))
    if uinput == 1:
        marks = int(input("Enter marks: "))
        if marks > 100 and marks < 0:
            print("Invalid marks")
            print("")
        elif marks >= 90 and marks <= 100:
            print(f"The grade for {marks} is A.")
            print("")
        elif marks >= 87 and marks <= 89:
            print(f"The grade for {marks} is A -. ")
            print("")
        elif marks >= 84 and marks <= 86:
            print(f"The grade for {marks} is B+.")
            print("")
        elif marks >= 80 and marks <= 83:
            print(f"The grade for {marks} is B.")
            print("")
        elif marks >= 77 and marks <= 79:
            print(f"The grade for {marks} is B -. ")
            print("")
        elif marks >= 74 and marks <= 76:
            print(f"The grade for {marks} is C+.")
            print("")
        elif marks >= 70 and marks <= 73:
            print(f"The grade for {marks} is C.")
            print("")
        elif marks >= 67 and marks <= 69:
            print(f"The grade for {marks} is C -. ")
            print("")
        elif marks >= 64 and marks <= 66:
            print(f"The grade for {marks} is D+.")
            print("")
        elif marks >= 62 and marks <= 63:
            print(f"The grade for {marks} is D.")
            print("")
        elif marks >= 60 and marks <= 61:
            print(f"The grade for {marks} is D -. ")
            print("")
        elif marks >= 0 and marks <= 59:
            print(f"The grade for {marks} is F.")
            print("")

    elif uinput == 2:
        print("Goodbye!")
        break
